Tests/TestingApps/PaywallsTester/PaywallsTester/Views/AppContentView.swift

```diff
@@ -168,13 +170,14 @@ private struct ProminentButton: View {
         Button(action: self.action) {
             Text(self.title)
                 .bold()
-                .padding()
                 .frame(maxWidth: .infinity)
-                .background(background)
-                .foregroundColor(.white)
-                .cornerRadius(10)
         }
+        .buttonStyle(.borderedProminent)
```

Comment:
This implementation broke in `visionOS`, so I simplified it using this SwiftUI API.